### DISCUSSION TITLE
Is this project abandoned?

### DISCUSSION THREAD

**Discussion by michalpolka** [mgmeyers/obsidian-kanban#1074] [Releases]
I was looking through all the open issues and commits and realised that there wasn't much (any?) activity from the owner recently. I then checked the releases and noticed that since May 2024, there haven't been any new releases, whereas previously they were very frequent.

I also can't seem to find anyone asking this question.

Is this project abandoned? If so, does anyone know if it's going to be picked up later/by someone else?

Also noticed that the owner is studying, per their bio, so it's very likely that they simply don't have the time. Anyway, I hope everything is ok!

**Comment by bcwilsondotcom** [comment]
It appears to be. Looks like the development roadmap has been removed and no PRs have been merged since May. Someone correct me if I'm wrong.

**Comment by danielgsanz** [comment]
This plugin is so helpful. I hope the development could be resumed anytime soon 🥺

**Comment by MLGHerobrine** [comment]
I emailed the owner of the repository asking if it's still maintained. I'll let you know if I receive a response.

**Comment by bcwilsondotcom** [comment]
@MLGHerobrine guessing you didn't get a response?

**Reply by MLGHerobrine** [reply]
@bcwilsondotcom Nope, sorry.

**Comment by GitMurf** [comment]
I believe the maintainer @mgmeyers has been working for the Obsidian team for a while now, right? This is probably the reason. But maybe that is good news for the new Bases feature as maybe he will work on the Kanban view for Bases :-)

**Comment by MrDaisyBates** [comment]
I'm assuming all the silence and delays here kinda confirms this plugin is abandoned?

It's def a shame. The ability to do great boards in Obsidian is very refreshing, and some of those roadmap things (like swimlanes, which could enable story mapping, as well) are really welcomed and underrepresented in the free/open source world.

**Reply by yousecjoe** [reply]
The only shame is on us now for not doing more for ourselves with the power we have been granted by LLMs. 
Ask your LLM to implement the features and fixes you desire. 
GPT-5.6-Sol can knock these features/bugs out in a day or less.
